test/judges/test-github-events.rb

```diff
@@ -240,6 +240,72 @@ def test_skip_event_when_user_equals_pr_author
     assert_nil(f[1])
   end
 
+  def test_skip_issue_event
+    WebMock.disable_net_connect!
+    stub_request(:get, 'https://api.github.com/repos/foo/foo').to_return(
+      body: { id: 42, full_name: 'foo/foo' }.to_json, headers: {
+        'content-type': 'application/json'
+      }
+    )
+    stub_request(:get, 'https://api.github.com/repositories/42').to_return(
+      body: { id: 42, full_name: 'foo/foo' }.to_json, headers: {
+        'content-type': 'application/json'
+      }
+    )
+    stub_request(:get, 'https://api.github.com/repositories/42/events?per_page=100').to_return(
+      body: [
+        {
+          id: '40623323541',
+          type: 'IssuesEvent',
+          public: true,
+          created_at: '2024-07-31 12:45:09 UTC',
+          actor: {
+            id: 42,
+            login: 'yegor256',
+            display_login: 'yegor256',
+            gravatar_id: '',
+            url: 'https://api.github.com/users/yegor256'
+          },
+          repo: {
+            id: 42,
+            name: 'yegor256/judges',
+            url: 'https://api.github.com/repos/yegor256/judges'
+          },
+          payload: {
+            action: 'opened',
+            issue: {
+              number: 1347,
+              state: 'open',
+              title: 'Found a bug',
+              body: "I'm having a problem with this."
+            }
+          }
+        }
+      ].to_json,
+      headers: {
+        'content-type': 'application/json'
+      }
+    )
+    stub_request(:get, 'https://api.github.com/user/42').to_return(
+      body: { id: 42, login: 'torvalds' }.to_json, headers: {
+        'content-type': 'application/json'
+      }
+    )
+    stub_request(:get, 'https://api.github.com/repos/yegor256/judges/issues/1347').to_return(
+      status: 200,
+      body: { number: 1347, state: 'open' }.to_json,
+      headers: { 'content-type': 'application/json' }
+    )
+    fb = Factbase.new
+    op = fb.insert
+    op.what = 'issue-was-opened'
+    op.repository = 42
+    op.issue = 1347
+    load_it('github-events', fb)
+    f = fb.query('(eq what "issue-was-opened")').each.to_a
+    assert_equal(1, f.length)
+  end
```

Comment:
@tank-bohr Maybe need add test for `issue-was-closed` case

Reply:
@tank-bohr I agree with @Yegorov. We need a test for `issue-was-closed` case